source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.pruners.HyperbandPruner.po

```diff
@@ -34,25 +34,30 @@ msgid ""
 ":math:`B \\over n`. `Hyperband "
 "<http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_ attacks this "
 "trade-off by trying different :math:`n` values for a fixed budget."
-msgstr ""
+msgstr "由于 SuccessiveHalving (SHA) 需要参数配置的个数作为其超参数. 对于一个给定的有限预算 :math:`B`, "
+"所有的配置平均下来有 :math:`B \\over n` 的资源. 所以, 这里存在一个 :math:`B` 和"
+":math:`B \\over n` 的权衡取舍. `Hyperband "
+"<http://www.jmlr.org/papers/volume18/16-558/16-558.pdf>`_ 通过为固定的预算设置不同的 :math:`n` 来解决这个权衡问题."
 
 #: of optuna.pruners._hyperband.HyperbandPruner:11
 msgid ""
 "In the Hyperband paper, the counterpart of "
 ":class:`~optuna.samplers.RandomSampler` is used."
-msgstr ""
+msgstr "在 Hyperband 论文中采用的是 :class:`~optuna.samplers.RandomSampler` 的对应物."
 
 #: of optuna.pruners._hyperband.HyperbandPruner:13
 msgid "Optuna uses :class:`~optuna.samplers.TPESampler` by default."
-msgstr ""
+msgstr "默认情况下 Opuna 使用 :class:`~optuna.samplers.TPESampler`."
 
 #: of optuna.pruners._hyperband.HyperbandPruner:14
 msgid ""
 "`The benchmark result "
 "<https://github.com/optuna/optuna/pull/828#issuecomment-575457360>`_ "
 "shows that :class:`optuna.pruners.HyperbandPruner` supports both "
 "samplers."
-msgstr ""
+msgstr "`基准测试 <https://github.com/optuna/optuna/pull/828#issuecomment-575457360>`_ "
+"表明 :class:`optuna.pruners.HyperbandPruner` 支持两种 sampler."
```

Comment:
"表明 :class:`optuna.pruners.HyperbandPruner` 对这两种 sampler都支持."

I think this can be more clear. The original version sounds like " :class:`optuna.pruners.HyperbandPruner` supports two samplers. "